node/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "azure-pipelines-task-lib",
-  "version": "2.11.0",
+  "version": "3.0.0",
```

Comment:
What about fixes for tasks that are still on Node 6?

Reply:
Node 6 will remain on the 2.x version, which is why this is targeting releases/3.x instead of master. Both versions will be maintained until we migrate in the box tasks to node 14.

Reply:
If we're breaking compat we should probably include a writeup somewhere in the repo that describes the upgrade steps needed

Reply:
Also, should probably mention that currently any changes should go into both master and this branch

Reply:
Oh, also we might want to release it as a preview version for now (3.0.0-preview) until we fully move away from our 2.x versions and are ready for it to just all be node 14 from here out